backend/routes/image.js

```diff
@@ -6,7 +6,7 @@ import { ValidationError } from '../errors';
 
 const routes = express.Router();
 
-routes.get('/images/:imageid', (req, res, next) => {
+routes.get('/:imageid', (req, res, next) => {
```

Comment:
Is it not better to explicitly keep "images" in this path for security reasons, preventing people from randomly retrieving information from entering a parameter without anything prior?

Reply:
/images/:imageid is still the route for this handler but moved the /images to `index.js` so that any routes created in image.js will have that prefix

Reply:
Ah ok that makes sense, all good then